dwitter/templates/dweet/dweet.html

```diff
@@ -39,6 +47,38 @@
           case "hideStats":
             setStatsVisibility(false);
             break;
+          case "record":
```

Comment:
rename to startGifRecording or something similar. (camelCase to be consistent with hideStats, I guess)